client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -202,8 +193,10 @@ const InstructionsRenderer = ({ testResult, testPageUrl, at }) => {
 
     const assertionsContent = parseListContent(assertions);
 
+    const Heading = `h${headingLevel}`;
```

Comment:
I didn't know that you could create HTML elements with strings in JSX like this. Very neat.